gap/perm/largebase.gi

```diff
@@ -507,6 +507,14 @@ end;
 #! <M>r \cdot k > 1</M>
 #! and <M>2 \cdot r \cdot k^2 \le n</M>.
 #!
+#! A large primitive group <M>H</M>of the above type which does have fixed
+#! points is handled as follows.  After a call of <C>LargeBasePrimitive</C>,
+#! the group <M>H</M> knows that it is primitive. Thus, after
+#! LargeBasePrimitive returns <K>NeverApplicable</K>, the function
+#! <Ref Func="ThrowAwayFixedPoints"/> will prune away the fixed points and set
+#! up a homomorphism. <C>LargeBasePrimitive</C> is then applicable to the image
+#! of that homomorphism.
```

Comment:
This phrasing really confused me. First yo talk about what happens if `LargeBasePrimitive` succeeds; and then you switch to what happens if it is not applicable; in that case it might be that `IsPrimitive` is false; so why then does it imply that `ThrowAwayFixedPoints` will prune away fixed points ?!?! DId you perhaps mean to refer to the return value "TemporaryFailure" ?

Anyway, it all got a bit clearer when I looked at the code and saw that `FindHomMethodsPerm.LargeBasePrimitive` simply *always* invokes `IsPrimitive` on any permutation group fed to it; OK, fair enough, the comment got a bit clearer. But I feel the comment should be made clear on its own.

I'd try to suggest something, but I am not exactly sure what you wanted to express here, so it's a bit difficult. But perhaps something simple: like this:

```suggestion
#! A large primitive group <M>H</M>of the above type which does have fixed
#! points is handled as follows:  After the first call to <C>LargeBasePrimitive</C>,
#! the group <M>H</M> knows whether it is primitive or not. Thus, if
#! LargeBasePrimitive returns <K>TemporaryFailure</K> for a primitive group, the function
#! <Ref Func="ThrowAwayFixedPoints"/> will use the new information about <M>H</M>
#! being primitive, and then is guaranteed to prune away the fixed points and set
#! up a reduction homomorphism. <C>LargeBasePrimitive</C> is then applicable to the image
#! of that homomorphism.
```

Reply:
Ok, I'll have a look before we merge this in.